lib/Monocle_Alternate.php

```diff
@@ -44,6 +45,21 @@ public function get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $
             $pkmn_in = substr($pkmn_in, 0, -1);
             $conds[] = "pokemon_id NOT IN ( $pkmn_in )";
         }
+        $float = $db->info()['driver'] == 'pgsql' ? "::float" : "";
+        if (!empty($miniv) && !is_nan((float)$miniv) && $miniv != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '((atk_iv + def_iv + sta_iv) / 45)' . $float . ' * 100 >= ' . $miniv;
+            } else {
+                $conds[] = '(((atk_iv + def_iv + sta_iv) / 45)' . $float . ' * 100 >= ' . $miniv . ' OR pokemon_id IN(' . $exminiv . ') )';
+            }
+        }
+        if (!empty($minlevel) && !is_nan((float)$minlevel) && $minlevel != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '(level >= ' . $minlevel;
+            } else {
+                $conds[] = '(level >= ' . $minlevel . ' OR pokemon_id IN(' . $exminiv . ') )';
+            }
+        }
 
```

Comment:
Monocle/Alternate fork now also has `get_active_by_id()`

Reply:
Duplicated this function into alternate